aqua/emacs-mac-app/Portfile

```diff
@@ -182,7 +184,12 @@ variant metal description {Enable experimental Metal support} {
     configure.args-append  --with-mac-metal
 }
 
-default_variants-append +nativecomp +treesitter
+variant renameapp description {Rename app bundle to avoid conflict with emacs-app} {
```

Comment:
Please humor my bikeshedding. Precedents for this kind of variant include:

- xpdf: no_mangle_names (has the opposite effect of this)
- afscompress: original_name
- minio-mc: altname

I kind of like the no_mangle_names scheme, where the variant would *not* be a default, and it would *introduce* the conflict instead of remove it.

What do you think?

Reply:
I have no strong preference for either option; my rationale was that it would be better to be explicit about the (possibly unexpected) changes introduced by MacPorts. Let's go with the no_mangle_names scheme since you like that one. Do you have a preference for the exact name of the variant? We could have `norename`, `norenameapp`, or versions of those with underscores. For the description, something like "Install as Emacs.app instead of EmacsMac.app" seems to convey the most information succinctly.

Reply:
I happened to be aware of no_mangle_names, and I was searching for a standard naming convention for this sort of variant, but having surveyed the field it seems there is no such standard. "Name mangling" is a well known concept but it seems the context is usually symbols in compiled objects, not a binary name, so if anything it introduces ambiguity here (does the variant modify the compiled content of Emacs?).

So on second thought I think your current formulation (renameapp being a default variant) is the clearest option.

Thanks for humoring me.